Gemfile

```diff
@@ -18,6 +18,7 @@ group :development do
 end
 
 gem 'acts_as_tenant'
+gem 'discard', '~> 1.0'
```

Comment:
@syncrou @lindgrenj6 Any thoughts on if we should pull in master or the latest SHA?  There have been a number of commits since it was release (most of the readme updates), not sure if any of these would effect us much.

Newer commits: https://github.com/jhawthorn/discard/compare/v1.0.0...master

Reply:
Oh I never thought about that, but thats quite a bit of difference. I'll get that changed so we're using master instead.